### DISCUSSION TITLE
MSC link adaption

### DISCUSSION THREAD

**Discussion by ThorisoMakhosa** [srsran/srsRAN_4G#1178] [General]
Hi all. I understand that from the srsRAN documentation that srsRAN does not support link adaption. But is there a way to know from the log files which DL and UL MSC one is using?

Thanks.

**Comment by ismagom** [comment]
MCS link adaptation is supported in 4G and 5G. If you are looking for 5G, please use https://github.com/srsran/srsRAN_Project
